README.md

```diff
@@ -73,6 +73,12 @@ Apply OVN DaemonSet and Deployment yamls.
 # Create OVN namespace, service accounts, ovnkube-db headless service, configmap, and policies
 kubectl create -f $HOME/work/src/github.com/ovn-org/ovn-kubernetes/dist/yaml/ovn-setup.yaml
 
+# Define egressips.k8s.ovn.org CRD
+kubectl create -f $HOME/work/src/github.com/ovn-org/ovn-kubernetes/dist/yaml/k8s.ovn.org_egressips.yaml
+
+# Define egressfirewalls.k8s.ovn.org CRD
+kubectl create -f $HOME/work/src/github.com/ovn-org/ovn-kubernetes/dist/yaml/k8s.ovn.org_egressips.yaml
```

Comment:
@roytman copy-and-paste error? the yaml file name needs a fix.

Reply:
indeed copy-paste, fixed.